core/src/main/java/org/hisp/dhis/android/core/domain/aggregated/data/AggregatedD2Progress.kt

```diff
@@ -25,48 +25,54 @@
  *  (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
  *  SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
+package org.hisp.dhis.android.core.domain.aggregated.data
 
-package org.hisp.dhis.android.core.tracker.exporter;
+import org.hisp.dhis.android.core.arch.call.D2Progress
+import org.hisp.dhis.android.core.arch.call.D2ProgressStatus
 
-import androidx.annotation.NonNull;
+data class AggregatedD2Progress(
+    @get:JvmName("getIsComplete")
+    override val isComplete: Boolean = false,
+    override val totalCalls: Int? = null,
+    override val doneCalls: List<String?> = emptyList(),
```

Comment:
I have the feeling this `List<String?>` could be `List<String>`. DataSet uids should only be added when they are synced.

Reply:
Maybe it is the url, I am not sure. Anyway, it looks like it shouldn't be null